common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1423,28 +1419,28 @@
-  "feature.wynntils.worldWaypointDistance.horizontalBoundingDistance.name": "Side Indicator Distance",
-  "feature.wynntils.worldWaypointDistance.maxWaypointTextDistance.description": "At what distance should the waypoint text stop rendering?",
-  "feature.wynntils.worldWaypointDistance.maxWaypointTextDistance.name": "Maximum Distance for Waypoint Text",
-  "feature.wynntils.worldWaypointDistance.name": "Waypoint Distance",
-  "feature.wynntils.worldWaypointDistance.scale.description": "How large should the marker be?",
-  "feature.wynntils.worldWaypointDistance.scale.name": "Scale",
-  "feature.wynntils.worldWaypointDistance.showAdditionalTextInWorld.description": "Whether the name of compass target should be displayed",
-  "feature.wynntils.worldWaypointDistance.showAdditionalTextInWorld.name": "Show name in world",
-  "feature.wynntils.worldWaypointDistance.showAdditonalTextAbove.description": "Whether the additional text should be displayed above the icon instead of bottom",
-  "feature.wynntils.worldWaypointDistance.showAdditonalTextAbove.name": "Show Additional Text Above",
-  "feature.wynntils.worldWaypointDistance.textColor.description": "What color should distance text be?",
-  "feature.wynntils.worldWaypointDistance.textColor.name": "Distance Text Color",
-  "feature.wynntils.worldWaypointDistance.textShadow.description": "What should the text shadow look like?",
-  "feature.wynntils.worldWaypointDistance.textShadow.name": "Text Shadow",
-  "feature.wynntils.worldWaypointDistance.topBoundingDistance.description": "How far should off-screen indicator be from top of the screen?",
-  "feature.wynntils.worldWaypointDistance.topBoundingDistance.name": "Top Indicator Distance",
+  "feature.wynntils.worldMarkers.bottomBoundingDistance.description": "How far should off-screen indicator be from bottom of the screen?",
+  "feature.wynntils.worldMarkers.bottomBoundingDistance.name": "Bottom Indicator Distance",
+  "feature.wynntils.worldMarkers.description": "Adds the ability to see the distance to your waypoint in the world.",
+  "feature.wynntils.worldMarkers.horizontalBoundingDistance.description": "How far should off-screen indicator be from two sides of the screen?",
+  "feature.wynntils.worldMarkers.horizontalBoundingDistance.name": "Side Indicator Distance",
+  "feature.wynntils.worldMarkers.iconRenderScale.description": "How large should the marker's icon be?",
+  "feature.wynntils.worldMarkers.iconRenderScale.name": "Icon Rendering Scale",
+  "feature.wynntils.worldMarkers.markerBeaconColor.description": "What default color should the beacon beam have?",
+  "feature.wynntils.worldMarkers.markerBeaconColor.name": "Marker Beacon Beam Color",
+  "feature.wynntils.worldMarkers.maxMarkerDistance.description": "At what distance should the waypoint text stop rendering?",
+  "feature.wynntils.worldMarkers.maxMarkerDistance.name": "Maximum Distance for Waypoint Text",
+  "feature.wynntils.worldMarkers.name": "Waypoint Distance",
```

Comment:
This does not seem like an adequate name and description of the feature. "World Markers and Beacons"? Or just "World Markers" and have the description as "Adds the ability to show in-world markers and beacons to certain map locations"